runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -5158,10 +5165,10 @@ class INTERPRETER_CLASS
 			returnStorage = (UDATA *)(UDATA)*(I_64 *)(_sp + 5); /* returnStructMemAddr */
 		}
 
-#if JAVA_SPEC_VERSION >= 20
+#if JAVA_SPEC_VERSION >= 21
 		/* The native memory is allocated at java level to save the execution state after performing the downcall. */
 		returnState = (I_32 *)(UDATA)*(I_64 *)(_sp + 7); /* returnStateMemAddr */
-#endif /* JAVA_SPEC_VERSION >= 20 */
+#endif /* JAVA_SPEC_VERSION >= 21 */
 
 #if JAVA_SPEC_VERSION >= 21
```

Comment:
Please remove the `#endif` / `#if` pair - the conditions are the same.

Reply:
Removed as suggested above.